lib/lintbot.js

```diff
@@ -5,7 +5,6 @@ const FetchParallel = require('./helpers/FetchParallel');
 const invariant = require('./helpers/invariant');
 const TimeoutError = require('./errors/TimeoutError');
 const FetchError = require('./errors/FetchError');
```

Comment:
:stop_sign: **Lint Error** Expected 1 empty line after require statement not followed by another require.

```suggestion
const FetchError = require('./errors/FetchError');

```

[import/newline-after-import](https://www.google.com/search?btnI&q=eslint%20rule%20import%2Fnewline-after-import)